user-console/source/org/pentaho/mantle/public/themes/crystal/mantleCrystal.css

```diff
@@ -1283,10 +1283,19 @@ a:active, a:hover {
 }
 
 .users-roles-list {
-  height: 432px;
+  height: 100%;
   width: 200px;
 }
 
+/* targetting IE 10 only */
+.IE10 .pentaho-tab-panel {
+    height:541px!important;
```

Comment:
@pamval I realize it's a bad option, but in this case it's a very specific scenario for a specific style to be overriden only in IE10. If even so it's not acceptable, I'll try to find another solution.